theano/gpuarray/opt.py

```diff
@@ -81,7 +81,7 @@ def query(self, *tags, **kwtags):
 gpu_seqopt.register('gpuarray_graph_optimization', GraphToGPUDB(), -0.5,
                     'fast_compile', 'fast_run', 'gpuarray')
 
-gpu_seqopt.register('gpuarray_local_optimiziations', gpu_optimizer, 1,
+gpu_seqopt.register('gpuarray_local_optimizations', gpu_optimizer, 1,
```

Comment:
thanks for the fix. To prevent regression in case other people used that bad sentences, can you add after 'gpuarray' the old string?

Thanks